tox.ini

```diff
@@ -70,6 +70,7 @@ envlist =
 # gevent 1.0 is not python 3 compatible
     {py27}-gevent{10}
     {py27,py34,py35,py36}-httplib
+    {py27,py34,py35,py36}-influx{50,51,52}
```

Comment:
Based on the comment below, this would be changed to `influx_contrib-{py27,py34,py35,py36}-influx{50,51,52}`.